rp2040-hal/src/timer.rs

```diff
@@ -292,7 +292,7 @@ pub trait Alarm: Sealed {
     /// called, this will trigger interrupt whenever this timestamp is reached.
     ///
     /// The RP2040 is unable to schedule an event taking place in more than
-    /// `u32::max_value()` microseconds.
+    /// `u32::MAX` microseconds.
```

Comment:
This is already covered by https://github.com/rp-rs/rp-hal/pull/802

Reply:
Nice let's merge it then and I'll drop that commit 👍